flatfs.go

```diff
@@ -79,14 +100,24 @@ type Datastore struct {
 	// sychronize all writes and directory changes for added safety
 	sync bool
 
-	diskUsage           int64
-	diskUsageCheckpoint int64
+	diskUsage   int64
+	storedValue diskUsageValue
+	// updateLock must be held when updating storedValue or writing
+	// to a file, it doesn't need to be held when reading
```

Comment:
to "a file"?

Reply:
I mean when writing `diskUsage` to a file.